lib/src/core/policy_engine/default_policy.rego

```diff
@@ -0,0 +1,43 @@
+package policy
+
+import future.keywords.every
+
+default allow = false
+
+allow {
+	every k, v in input {
+		judge_field(k, v)
+	}
+}
+
+judge_field(input_key, input_value) {
+	has_key(data.reference, input_key)
+	reference_value := data.reference[input_key]
+	match_value(reference_value, input_value)
+}
+
+judge_field(input_key, input_value) {
+	not has_key(data.reference, input_key)
+}
+
+match_value(reference_value, input_value) {
+	not is_array(reference_value)
+	input_value == reference_value
+}
+
+match_value(reference_value, input_value) {
+	is_array(reference_value)
+	array_include(reference_value, input_value)
+}
+
+array_include(reference_value_array, input_value) {
+	reference_value_array == []
```

Comment:
So any claim will be true if it is made against a reference array that is empty? This is what we do in `simple-kbs` as well (because it's hard to define a good wildcard value). Let's just make sure we document this clearly because it could lead to accidental policy validation.

Reply:
Yes, this default policy needs to be written into documents to explain to users, but now is not the time because AS is still in the early construction stage. Let's remember it and explain this when adding documents to AS.